src/plugins/Github3/GitHub3Plugin.cs

```diff
@@ -67,19 +69,26 @@ public static string OAuthToken
 
     [Export(typeof(IGitPlugin))]
     [Export(typeof(IRepositoryHostPlugin))]
-    public class GitHub3Plugin : GitPluginBase, IRepositoryHostPlugin
+    [Export(typeof(IGitPluginForCommit))]
+    public class GitHub3Plugin : GitPluginBase, IRepositoryHostPlugin, IGitPluginForCommit
     {
+        private const int MaxMostRecentIssuesDisplayed = 10;
```

Comment:
I suggest not to hard code it but to make it a configurable. If you don't want to expose it at the settings dialog then make it a hidden setting configurable through the config file directly for super-advanced users.

Reply:
Put as setting as it is easier.